x/cdp/migrations/v2/store_test.go

```diff
@@ -0,0 +1,54 @@
+package v2_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/cosmos-sdk/simapp"
+	"github.com/cosmos/cosmos-sdk/testutil"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	v2cdp "github.com/kava-labs/kava/x/cdp/migrations/v2"
+	"github.com/kava-labs/kava/x/cdp/types"
+)
+
+func TestStoreMigrationAddsKeyTableIncludingNewParam(t *testing.T) {
+	encCfg := simapp.MakeTestEncodingConfig()
+	cdpKey := sdk.NewKVStoreKey(types.ModuleName)
+	tcdpKey := sdk.NewTransientStoreKey("transient_test")
+	ctx := testutil.DefaultContext(cdpKey, tcdpKey)
+	paramstore := paramtypes.NewSubspace(encCfg.Codec, encCfg.Amino, cdpKey, tcdpKey, types.ModuleName)
+
+	// Check param doesn't exist before
+	require.False(t, paramstore.Has(ctx, types.KeyBeginBlockerExecutionBlockInterval))
+
+	// Run migrations.
+	err := v2cdp.MigrateStore(ctx, paramstore)
+	require.NoError(t, err)
+
+	// Make sure the new params are set.
+	require.True(t, paramstore.Has(ctx, types.KeyBeginBlockerExecutionBlockInterval))
```

Comment:
Let's also add an assertion that the value is what we expect.  The migrator is also ensuring the default value is set but we don't have a test asserting it does -- only that it sets the value to something.